### PR TITLE
Story4 reaper

### PR DESCRIPTION
**Pull request by tmhnsly**
Please wait for code review to be merged in before final pull request

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 let ghost = document.querySelector('.ghost');
 let pumpkin = document.querySelector('.pumpkin');
 let reaper = document.querySelector('.grim_reaper');
-let lineSeven = document.querySelector('.line_seven');
+let lineEight = document.querySelector('.line_eight');
 
 let start = document.querySelector('.start_area');
 let collision = document.querySelectorAll('.collision');
```
Comment (ellie-r) [inline script.js]: in the code line 24 is missing a semi-colon

Comment (ellie-r) [reply script.js]: and line 37

Comment (ellie-r) [reply script.js]: and line 48

Comment (ellie-r) [reply script.js]: and line 63

Comment (ellie-r) [reply script.js]: 73, 86, 97, 103, 116, 132

**Review by ellie-r — CHANGES_REQUESTED** [review]
all good just missing some semicolons

**Review by ellie-r — APPROVED** [review]
approved

File: script.js [hunk 2]
```diff
@@ -113,19 +107,21 @@ document.querySelector('.start-button').addEventListener('click', () => {
...
-//call functions from here
+//call main function to start game from here
 function runGame() {
     ghostMove.play();
     pumpkinMove.play();
-    reaperMove.play();
     listenForCollisions();
     listenForWinning();
-}
+    lineEight.addEventListener('mouseleave',() => {
```
Comment (SarahA2020) [inline script.js]: Stop this animation on win or loss

**Review by SarahA2020 — CHANGES_REQUESTED** [review]

**Review by SarahA2020 — APPROVED** [review]

**Pull request merged**